extensions/sequelize/src/types.ts

```diff
@@ -3,6 +3,18 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+declare module '@loopback/repository' {
+  interface Inclusion {
+    /**
```

Comment:
Why does it need to affect the types of core repository module, instead of type extension in the Sequelize repository itself. For the respective methods (`find()`, `findOne()`, and `findById()`).

I'm not certain about the behaviour this code will have but It can reflect to non sequelize repositories as well, which might not be intended here.

Reply:
@shubhamp-sf Thanks for reviewing this. Yes, the TS declaration merging approach affects non-sequelize repositories too. Extending the method interfaces is much better, however, it doesn't cover the relation query "find" case like:
```ts
myRepository(foreignKey).find({
  include: [
    {
      // ...
      required: true
    }
  ]
}
```

I tried a couple of approaches for that case. Perhaps defining new interfaces like "SequelizeHasManyRepositoryFactory" that extend the `find` method in the underlying interfaces such as `HasManyRepository`?